src/osp/Active/SysPrefabInit.h

```diff
@@ -0,0 +1,96 @@
+/**
+ * Open Space Program
+ * Copyright © 2019-2022 Open Space Program Project
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHEfR DEALINGS IN THE
+ * SOFTWARE.
+ */
+#pragma once
+
+#include "activetypes.h"
+#include "basic.h"
+#include "drawing.h"
+#include "physics.h"
+
+#include "../types.h"
+#include "../Resource/resourcetypes.h"
+
+#include <Corrade/Containers/ArrayView.h>
+
+namespace osp::active
+{
+
+/**
+ * @brief Data needed to initialize a Prefab
+ *
+ * This allows separate systems (doing physics, hierarchy, drawables, ...) to
+ * work in arallel on initializing a prefab.
+ *
+ * Intended to be created and quickly destroyed once the prefab is created.
+ * This is often the span of a single frame.
+ */
+struct TmpPrefabInit
+{
+    osp::ResId m_importerRes;
```

Comment:
```
Variable 'osp::active::TmpPrefabInit::m_importerRes' is uninitialized. Always initialize a member variable (type.6).

Warning
Variable 'osp::active::TmpPrefabInit::m_importerRes' is uninitialized. Always initialize a member variable (type.6).
Show more details 
```

If you're going to initialize m_pTransform, then you should initialize all of them.